### PR TITLE
Provide a license

### PR DESCRIPTION
**Pull request by mb1986**

### CODE REVIEW

**Comment by beelux** [comment]
> You can license the patches separately from the rest of the code. I would recommend using the MIT license for everything but the patches, and keeping those as all rights reserved or something.
> [(Link to original comment)](https://github.com/mb1986/rm-hacks/issues/1#issuecomment-1783464560)

@Eeems Do you believe this to be sufficient?
Technically you have a contribution (as small as it might be) while there was no proper license, thus I belive you should be asked too in regards to licensing.

I'm fine with it, but I think the disclaimer could use a "not affiliated with reMarkable AS in any way", as ddvk put it.

**Comment by Eeems** [comment]
It might be worth putting a small readme on the patches folder that reiterates the statement from the readme.

File: README.md [hunk 1]
```diff
@@ -80,3 +80,18 @@ This patch includes the following changes:
...
 - The toolbar icon changes from the default open icon to a brush icon.
 - The close icon (x) in the top right corner has been removed.
+
+## License
+
+This repository is provided under the MIT License, as described in the
+root `LICENSE` file. However, please note that all the files in the "patches"
+directory are marked as "All Rights Reserved," which means that they are
+protected by copyright, and no specific license information is available for
+them. Users should assume that they have no rights or permissions to use,
```
Comment (michael-lenz) [inline README.md:90]: This means that without express "permission to use [the patches]", people cannot install your bundle of hacks.

Also the phrasing - "should assume that" - is odd. 
I suggest

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+==============================================================================
+                            Important Licensing Note
+==============================================================================
+
+Please be aware that this repository contains a directory named "patches" with
+binary data files. These files are subject to a separate licensing arrangement
+and are marked as "All Rights Reserved." This means they are protected by
+copyright, and no specific license information is provided for them. Users
+should assume that they have no rights or permissions to use, modify, or
```
Comment (michael-lenz) [inline LICENSE:31]: This means that without express "permission to use [the patches]", people cannot install your bundle of hacks.

Also the phrasing - "should assume that" - is odd. 
I suggest: "Users are not allowed to modify or distribute these files without explicit permission from the copyright holder."

Beware, tho, IANAL

**Comment by mb1986** [comment]
Yes, I'll try to apply the suggestions and hopefully merge it this weekend.

**Comment by mb1986** [comment]
Sorry but it'll have to wait another day...

File: LICENSE (hunk 2 above)
Comment (mb1986) [reply LICENSE]: Thanks for the suggestion, I've just applied it.

File: README.md [hunk 3]
```diff
@@ -80,3 +80,18 @@ This patch includes the following changes:
   [ddvk's Qt Plugin for Lamy Pen](https://github.com/ddvk/remarkable-stylus) first. You will not need to install ddvk-hacks like it recommends, as this replaces that dependency.
 - The toolbar icon changes from the default open icon to a brush icon.
 - The close icon (x) in the top right corner has been removed.
+
+## License
+
+This repository is provided under the MIT License, as described in the
```
[the remark this answers is not in the document]
Comment (mb1986) [reply README.md]: Thanks for the suggestion, I've just applied it.

**Comment by Eeems** [comment]
lgtm

File: patches/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# rM Hacks / Patches
+
+All the files in the current directory are marked as "All Rights Reserved,"
+which means that they are protected by copyright, and no specific license
+information is available for them.  Users are not allowed to modify or
```
Comment (michael-lenz) [inline patches/README.md:5]: "Don't repeat yourself". Especially with license/contact terms it is important to have a single source of truth, the license file. I recommend likng it like in the root README. 

AFAIK stating a number of prohibitions (modification and distribution) does not mean that anything else (usage) is fair game, especially with "all rights reserved".
Hence maybe something like this: "Users are allowed to install and use these files via the installation script provided."

**Review by michael-lenz — CHANGES_REQUESTED** [review]

File: patches/README.md (hunk 4 above)
Comment (michael-lenz) [reply patches/README.md]: Sorry for being pedantic about DRY. 😅  I've been bitten by that one too many times..

That paragraph sounds good to me

**Review by michael-lenz — APPROVED** [review]

File: patches/README.md (hunk 4 above)
Comment (mb1986) [reply patches/README.md]: Don't be sorry, I really appreciate your inputs, thanks!

**Pull request merged**